### PR TITLE
feat(seat): Added Seat Icon

### PR DESCRIPTION
**Pull request by dhirajchawla2000**
This change is [Reviewable](https://reviewable.io/reviews/qantasairways/runway/54)

### CODE REVIEW

File: src/icons/SeatIcon/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { generate } from 'shortid';
+
+function SeatIcon({ height, width, className }) {
```
Comment (nnattawat) [inline src/icons/SeatIcon/index.js]: I reckon we should also allow consumer to pass on any props to svg. Something like:

```
SeatIcon({ height, width, className, ...otherProps }) {
  <svg {...otherProps} />
}
```

File: src/icons/SeatIcon/index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        <g mask={`url(#${maskId})`}>
+          <path d="M-5-2h24v24H-5z" />
+        </g>
+      </g>
+    </svg>
+  );
+}
+
+SeatIcon.propTypes = {
+  height: PropTypes.string,
```
Comment (nnattawat) [inline src/icons/SeatIcon/index.js]: Should height and width be `integer`

Comment (dhirajchawla2000) [reply src/icons/SeatIcon/index.js]: I have kept it similar to other icons in the repo

File: src/icons/SeatIcon/index.js (hunk 1 above)
Comment (dhirajchawla2000) [reply src/icons/SeatIcon/index.js]: I have kept it similar to other icons in the repo.

**Pull request merged**